docs/spec/State-Machine-Behavior-Elements/Junction-Definitions.adoc

```diff
@@ -42,7 +42,10 @@ state machine Device {
 
   junction J1 {
     if coldStart enter OFF \
-    else do initPower enter ON
+    else do {
+      initPower
+    } 
```

Comment:
We need to close up the line between 47 and 48 or add a backslash. Similarly elsewhere.

Reply:
Fixed